juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule

```diff
@@ -86,7 +86,7 @@ healthbot {
                 enumerate Full as 1;
             }
             trigger l3vpn-pe-interface-state {
-                alert-type vpn.l3vpn.state.l3vpn-pe-interface-state;			
```

Comment:
I think we should keep this as is and not include protocol under alert type. For all the new alerts also same alert-type should be used.